src/webpack/__tests__/test.js

```diff
@@ -116,6 +115,17 @@ describe('zacs-loader', () => {
     const js = modules.find(m => m.rawRequest === './examples/brokenWhitespace.js')._source._value
     expect(js).toMatchSnapshot()
   })
+  it(`has configurable cache directory and extension`, async () => {
+    const stats = await compile('examples/basic.js', {
+      cacheDirectory: path.resolve(__dirname, '.zacs-cache-test'),
+      extension: '.zacstest.css',
```

Comment:
Interesting, in theory you can spit out *.scss files and feed them to sass-loader, so you aren't limited to PostCSS exclusively

Reply:
yes, absolutely! Linaria does this so i just did the same